package.json

```diff
@@ -22,5 +22,8 @@
     "type": "git",
     "url": "https://github.com/NetherlandsForensicInstitute/hansken-js.git"
   },
-  "devDependencies": {}
+  "devDependencies": {},
+  "dependencies": {
+    "express": "^4.17.1"
```

Comment:
Shouldn't this be a dev dependency? Looks like it's only used to run examples.

Reply:
True

Reply:
MOved